tests/unit/amo/components/ErrorPage/TestNotFound.js

```diff
@@ -53,4 +54,34 @@ describe(__filename, () => {
     expect(landingLinks.at(0)).toHaveProp('to', '/extensions/');
     expect(landingLinks.at(1)).toHaveProp('to', '/themes/');
   });
+
+  it('handles a localized string with links inverted', () => {
```

Comment:
I'm not a fan of regression tests (when they're redundant) because they slow down the test suite ☠️  but it's not a big deal.